flow/Tracing.actor.cpp

```diff
@@ -101,9 +102,11 @@ struct TraceRequest {
 
 // A server listening for UDP trace messages, run only in simulation.
 ACTOR Future<Void> simulationStartServer() {
-	TraceEvent(SevInfo, "UDPServerStarted").detail("Port", FLOW_KNOBS->TRACING_UDP_LISTENER_PORT);
-	state NetworkAddress localAddress =
-	    NetworkAddress::parse("127.0.0.1:" + std::to_string(FLOW_KNOBS->TRACING_UDP_LISTENER_PORT));
+	TraceEvent(SevInfo, "UDPServerStarted")
+	    .detail("Address", FLOW_KNOBS->TRACING_UDP_LISTENER_ADDR)
+	    .detail("Port", FLOW_KNOBS->TRACING_UDP_LISTENER_PORT);
+	state NetworkAddress localAddress = NetworkAddress::parse(FLOW_KNOBS->TRACING_UDP_LISTENER_ADDR + ":" +
```

Comment:
I think this address should always be `127.0.0.1`, and below where `localAddress` is created, a check should be added to set the address to `127.0.0.1` if running in simulation. This way the simulation tests will continue to correctly parse traces even if `TRACING_UDP_LISTENER_ADDR` is set to an external address.